unityfiles/Assets/Scripts/CharacterController2D.cs

```diff
@@ -0,0 +1,112 @@
+using UnityEngine;
+
+public class CharacterController2D : MonoBehaviour
+{
+	[SerializeField] private float m_JumpForce = 400f;							// Amount of force added when the player jumps.
+	[Range(0, 1)] [SerializeField] private float m_CrouchSpeed = .36f;			// Amount of maxSpeed applied to crouching movement. 1 = 100%
+	[Range(0, .3f)] [SerializeField] private float m_MovementSmoothing = .05f;	// How much to smooth out the movement
+	[SerializeField] private bool m_AirControl = false;							// Whether or not a player can steer while jumping;
+	[SerializeField] private LayerMask m_WhatIsGround;							// A mask determining what is ground to the character
+	[SerializeField] private Transform m_GroundCheck;							// A position marking where to check if the player is grounded.
+	[SerializeField] private Transform m_CeilingCheck;							// A position marking where to check for ceilings
+	[SerializeField] private Collider2D m_CrouchDisableCollider;				// A collider that will be disabled when crouching
```

Comment:
CrouchDisableCollider hva jør denne?
Betyr dette at man alltid har to collidere og denne blir disabled når man crotcher, hvis ja hvor er Crouch collideren?

Reply:
Ja, men dette gjelder hvis vi mener at characteren skal ha mulighet til å crouche